ci/taos/config/kas-files/kas-YOUR-PROJECT-sample.yml

```diff
@@ -0,0 +1,18 @@
+#
```

Comment:
Could you add new file as follows for maintenance and convenience?
```bash
$ cd .../kas-files/
$ vi README.md
This folder is created to run plugins-staging/pr-postbuild-build-yocto-kas.sh.
The YAML files are to be used as a configuration file.

Reply:
Great! I added a README.md